data/lib/core/position.lua

```diff
@@ -18,29 +18,29 @@ function Position:getNextPosition(direction, steps)
 	end
 end
 
-function Position:moveUpstairs()
-	local isWalkable = function (position)
-		local tile = Tile(position)
-		if not tile then
-			return false
-		end
+function Position:isWalkable()
+	local tile = Tile(position)
```

Comment:
Remove this line, use Tile:isWalkable() on line 21.
Then replace tile with self on all functions

ex (line 23)
```
if not self then
...
end
```